openapi3/issue638_test.go

```diff
@@ -0,0 +1,17 @@
+package openapi3
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestIssue638(t *testing.T) {
+	for i := 0; i < 50; i++ {
+		loader := NewLoader()
+		loader.IsExternalRefsAllowed = true
+		doc, err := loader.LoadFromFile("../openapi3/testdata/issue638/test1.yaml")
```

Comment:
Please add a not on this line about the path expression being important for this regression test.

Reply:
I added a comment about path expression.